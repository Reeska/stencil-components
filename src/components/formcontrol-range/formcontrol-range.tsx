import { Component } from "@stencil/core";


@Component({
  tag: 'evb-range',
  styleUrl: 'formcontrol-range.css',
  shadow: true
})
export class EvbRange {

  render() {
    return (
      <input type="range" />
    );
  }
}
