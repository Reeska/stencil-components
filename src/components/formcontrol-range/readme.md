# evb-range



<!-- Auto Generated Below -->


## Properties

| Property | Attribute | Description | Type     | Default     |
| -------- | --------- | ----------- | -------- | ----------- |
| `max`    | `max`     |             | `number` | `undefined` |
| `min`    | `min`     |             | `number` | `undefined` |
| `step`   | `step`    |             | `number` | `undefined` |
| `value`  | `value`   |             | `number` | `undefined` |


## Events

| Event       | Description | Type                  |
| ----------- | ----------- | --------------------- |
| `evbBlur`   |             | `CustomEvent<void>`   |
| `evbChange` |             | `CustomEvent<number>` |
| `evbFocus`  |             | `CustomEvent<void>`   |
| `evbInput`  |             | `CustomEvent<number>` |


----------------------------------------------

*Built with [StencilJS](https://stenciljs.com/)*
