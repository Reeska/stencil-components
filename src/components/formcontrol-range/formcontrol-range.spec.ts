import { EvbRange } from "./formcontrol-range";

describe('EvbRange', () => {
  let component: EvbRange;

  beforeEach(() => component = new EvbRange());

  it('should render', () => {
    expect(component.render()).toMatchSnapshot();
  });
});
