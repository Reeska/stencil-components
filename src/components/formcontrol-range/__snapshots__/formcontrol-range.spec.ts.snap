// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`EvbRange should render 1`] = `
Object {
  "elm": undefined,
  "ishost": false,
  "vattrs": Object {
    "max": undefined,
    "min": undefined,
    "onBlur": [Function],
    "onChange": [Function],
    "onFocus": [Function],
    "onInput": [Function],
    "step": undefined,
    "type": "range",
    "value": undefined,
  },
  "vchildren": null,
  "vkey": undefined,
  "vname": undefined,
  "vtag": "input",
  "vtext": undefined,
}
`;
